src/main/java/com/google/sps/data/BackgroundTaskManager.java

```diff
@@ -26,8 +30,21 @@ public void replaceFaultyInstances() {
     }
 
     @Override
-    public void deleteInactiveAttendees() {
-        throw new RuntimeException("Unimplemented");
+    public void deleteInactiveAttendees(DatastoreClientInterface datastore, String sessionId) { 
+        List<Attendee> attendees = datastore.getAttendeesInSession(sessionId);
+        for(int i = 0; i <= attendees.length(); i++){
+            AttendeeInterface currentAttendee = attendees.get(i);
+            AttendeeeInterface screenName = currentAttendee.getScreenName();
+            long timePolled = currentAttendee.getTimePolled();
```

Comment:
specify the units here, add Ms (I think they are in milliseconds?) to the ends of these long times

Reply:
Also I think you want currentAttendee.getTimePolled().getTime();